### PR TITLE
initial function generate-sequence

### PR DESCRIPTION
**Pull request by lubankp**
@lubankp added homework generate-sequence

### CODE REVIEW

**Comment by ziobron** [comment]
🏅 +1 XP for cooperation goes to @lubankp

**Comment by ziobron** [comment]
✅ CI Passed - Task: generate-sequence
🏅 Granted 5 XP for @lubankp

**Comment by ziobron** [comment]
⏱ 3 XP extra for @lubankp for on-time delivery

File: homework/generate-sequence/generateSequence.hpp [hunk 1]
```diff
@@ -2,6 +2,11 @@
 #include <vector>
 
 std::vector<int> generateSequence(int count, int step) {
-    // TODO: Implement me :)
-    return {};
+    
+    std::vector<int> vec;
+
+    for(auto element=1; element <= count; element++ ){
+        vec.push_back(step*element);
```
Comment (nauka-programowania-MA) [inline homework/generate-sequence/generateSequence.hpp:9]: Add space after `if`, `for`, `while` `function` etc...

Add space between operator like `+` `-` `/` `:` `&&` etc..

File: homework/generate-sequence/generateSequence.hpp [hunk 2]
```diff
@@ -2,6 +2,11 @@
 #include <vector>
 
 std::vector<int> generateSequence(int count, int step) {
-    // TODO: Implement me :)
-    return {};
+    
+    std::vector<int> vec;
```
Comment (nauka-programowania-MA) [inline homework/generate-sequence/generateSequence.hpp:6]: Reserve memory when you know how many elements will be inside vector.

**Review by nauka-programowania-MA — APPROVED** [review]

**Pull request closed without merging**